content/blogs/2024-04-05-getting-started-with-kestra.md

```diff
@@ -0,0 +1,315 @@
+
+```yaml
+- id: python_output
+  type: io.kestra.core.tasks.log.Log
+  message: "Number of stars: {{ outputs.python_script.vars.output }}"
+```
+
+Your new task should look like the following which will get out new output and print it out to the logs clearly for us to see. When we execute it, we should see it separated from all the Python logs for easier reading!
+
+![python_output](/blogs/2024-04-05-getting-started-with-kestra/python_output.png)
+
+## Adding a Notification
+
+Now we can take this one step further and send this output to a messaging app to notify us more easily on the number of stars, rather than digging through logs to find out the final value. For this example, we’ll use Discord but this will work with any of the Plugins in the Notifications group. 
+
+For this example, we can use the UI to build it rather than YAML as they’ll be a lot more customisable fields. When we edit our flow, we can open a view that shows YAML on one side, and the topology view on the other giving you the best of both worlds. Underneath the `python_output` task, we can press the ➕ to add a new task and search for Discord. 
+
+We’re going to use the `DiscordExecution` task as this lets us push a message to a webhook which will send a message to a channel. The other is useful if you want your flow to trigger based on an action inside of Discord. Now we’ve opened the `DiscordExecution` page, we’re presented with a long list of properties which can be overwhelming but we can focus on the required ones for now!
+
+![discord_ui](/blogs/2024-04-05-getting-started-with-kestra/discord_ui.png)
+
+For our Discord message, we’ll need to give this task an `id` , as well as a Webhook URL which we can get from Discord. While nothing else is required, there’s plenty of customisation to make the message feel more polished and clearer such as adding a title and avatar. For this example, we’ll call the task `send_notification` and change the username to be *Kestra*. We can also add an Avatar by using the URL of the GitHub Organisation profile picture. 
+
+Instead of hard coding this straight into the `avatarUrl` box, we can create an **input** to allow us to reuse this later on in case we send notifications to multiple platforms for example. Our input should look like the example below which we can put above the tasks in our flow, similar to what you would do with constants in Python.
+
+```yaml
+inputs:
+  - id: kestra_logo
+    type: STRING
+    defaults: https://avatars.githubusercontent.com/u/59033362?v=4
+```
+
+While we’re creating inputs, we can also make our Webhook URL an input incase we want to reuse it too. Discord has a [great guide](https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks) on how to generate the Webhook URL for a specific channel so all the messages are sent there. 
```

Comment:
```suggestion
While we’re creating inputs, we can also make our Webhook URL an input in case we want to reuse it, too. Discord has a [great guide](https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks) on how to generate the Webhook URL for a specific channel so all the messages are sent there. 
```